### DISCUSSION TITLE
Beta Versionen werden nicht mehr angezeigt

### DISCUSSION THREAD

**Discussion by stuckis6er** [SukramJ/aiohomematic#1162] [General]
Hallo, in HACS und GitHub werden mir keine Beta-Versionen mehr angezeigt obwohl ich über neue Betas per Email benachrichtigt wurde. 

Gruß Ralf

**Comment by SukramJ** [comment]
Manchmal werden die auch wieder gelöscht.
Aktuell ist keine vorhanden.
